WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1010,7 +1010,7 @@ public void onPhotoPickerIconClicked(@NonNull PhotoPickerIcon icon) {
     public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
         MenuInflater inflater = getMenuInflater();
-        if (mShowNewEditor || mShowAztecEditor) {
+        if (mShowNewEditor || mShowAztecEditor || mShowGutenbergEditor) {
```

Comment:
Code and logic look good - only thing I found is that these expressions:

- `mShowNewEditor || mShowAztecEditor || mShowGutenbergEditor`
and 
- `!mShowNewEditor && !mShowAztecEditor && !mShowGutenbergEditor`

appear several times (the first one 5 times, the second one 2 times)

Can we move these to functions like these?:

```
private boolean isShowAnyNewEditor() {
	return (mShowNewEditor || mShowAztecEditor || mShowGutenbergEditor);
}
```
```
private boolean isDontShowAnyNewEditor() {
	return (!mShowNewEditor && !mShowAztecEditor && !mShowGutenbergEditor);
}
```

Reply:
Makes sense 👍 . Added a shorthand method with 1047ab0695.

Only added one method since I think the second expression is just the logical negative of the first.